prow/cmd/hook/main.go

```diff
@@ -173,12 +173,26 @@ func main() {
 
 	pluginAgent := &plugins.PluginAgent{}
 
+	ownersClient := repoowners.NewClient(gitClient, githubClient, pluginAgent.MDYAMLEnabled)
+	ownersClient.DirBlacklist = func(org, repo string) []string {
+		conf := configAgent.Config().OwnersDirBlacklist
+
+		if list, ok := conf.Repos[org+"/"+repo]; ok {
+			return list
```

Comment:
For the other configurations (like plugins, for instance) we do the union of repo-specific and org-specific, right? Maybe it would be good to mirror that here

Reply:
Ok. Will do.